packages/server-core/src/hooks/verify-project-owner.ts

```diff
@@ -39,7 +40,7 @@ export default () => {
     if (context.params.isInternal) return context
     const loggedInUser = context.params.user as UserType
     if (!loggedInUser) throw new NotAuthenticated('No logged in user')
-    if (loggedInUser.scopes && loggedInUser.scopes.find((scope) => scope.type === 'admin:admin')) return context
+    if (loggedInUser.scopes && (await checkScope(loggedInUser, 'projects', 'write'))) return context
```

Comment:
@aditya-mitra, I think instead of `return context`, we should have `throw new Forbidden()` here.